core/rust/qdbr/parquet2/README.md

```diff
@@ -0,0 +1,158 @@
+# Parquet2
```

Comment:
They weren't stricly speaking necessary to build the library as a dependency.

I decided to bring the missing code in so we can continue to run `cargo test` on `parquet2` on CI (a future task) to lessen the risk we break the code as we continue editing it.

Now that the files are in place `cargo build` works again (it doesn't without the extra files) and the tests seem to work.

Reply:
Got it. I've also noticed formatting changes in the existing `parquet2` files. Do they come from the upstream repo or you've run formatting?

Reply:
I've run `cargo fmt` after I made some API fixes. Call it force of habit :-)

Reply:
I see. Potentially this could cause conflicts if we ever decide to merge upstream changes, but the parquet2 library seems abandoned, so it's not a big deal.